announcements/2021-Week-02/announcement.md

```diff
@@ -0,0 +1,18 @@
+# :zap: This Week in Provider-Independent Announcements (January 15, 2021):zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+[Prometheus Operator v0.5.0](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.5.0/) upgrades to the chart with Prometheus v2.22.1.
```

Comment:
@TheoBrigitte @glitchcrab Can you suggest a change if there are other customer-facing changes or improvements please?

Reply:
Notables changes are:

* prometheus-operator v0.42.1 -> v0.43.2: introduces a new AlertmanagerConfig CRD. https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.43.0
* prometheus v2.21.0 -> v2.22.1 : the experimental gRPC API v2 has been
removed. https://github.com/prometheus/prometheus/releases/tag/v2.22.0

Reply:
is this already included in the announcement and if not can you add it?

Reply:
I will add it